### PR TITLE
Update README to add initial data for PyCon India 2023

### LINKED ISSUE #16: Update PyCon India 2023 repo with all details

### PR DESCRIPTION
**Pull request by Sc0rpi0n101**
Fixes #16

### CODE REVIEW

**Comment by soumendrak** [comment]
@Sc0rpi0n101 

- Adding all the name contacts (zulip links?) would be better as hyperlinks.
- [Here is a source of Zulip links](https://docs.google.com/document/d/1Ya4r54g3Ellm2_ALvvCZZReVKxSBw4Di8LKX8ilLdFU/edit#heading=h.zigecab22kvt) of few names as a reference.

**Comment by Sc0rpi0n101** [comment]
Good Idea. Will do.

File: README.md [hunk 1]
```diff
@@ -1 +1,110 @@
...
+- Conference: 30th Sep - 1st Oct, 2023
+- DevSprint: 2nd Oct, 2023
+
+## Conference Structure
+- TBD
+
+## Working Groups
+
+* Conference Chair: [Sanchit Balchandani](https://pyconindia.zulipchat.com/#narrow/pm-with/231358-Sanchit-Balchandani)
+* Conference Co-Chairs:
```
Comment (inovizz) [inline README.md:16]: We have two co-chairs.

Kalyan Prasad
Snehith Allamraju

File: README.md [hunk 2]
```diff
@@ -1 +1,110 @@
...
+- Meeting:
+- Chair: [Soumendra](https://pyconindia.zulipchat.com/#narrow/pm-with/231777-Soumendra)
+- Shadow:
+- Zulip Stream: https://pyconindia.zulipchat.com/#narrow/stream/381471-wg.2Freview
+
+### Technology
+
+- Meeting:
+- Chair: [Bharat Saraswat](https://pyconindia.zulipchat.com/#narrow/pm-with/231146-Bharat-Saraswat)
+- Shadow:
```
Comment (inovizz) [inline README.md:39]: Shadow - Priyanshu Krishnan

File: README.md [hunk 3]
```diff
@@ -1 +1,110 @@
...
+- Zulip Stream: https://pyconindia.zulipchat.com/#narrow/stream/226092-wg.2Ftechnology
+
+### Promotions/Communications
+
+- Meeting:
+- Chair: [Bhushan Kapkar](https://pyconindia.zulipchat.com/#narrow/pm-with/608047-BHUSHAN-KAPKAR)
+- Shadow:
```
Comment (inovizz) [inline README.md:46]: Shadow - Amritha Madam

File: README.md [hunk 4]
```diff
@@ -1 +1,110 @@
...
+- Meeting:
+- Chair:
+- Shadow:
+- Zulip Stream: https://pyconindia.zulipchat.com/#narrow/stream/247766-wg.2Fdiversity
+
+### Design
+
+- Meeting:
+- Chair: [Aravind Sekar](https://pyconindia.zulipchat.com/#narrow/pm-with/270428-Aravind-Sekar)
+- Shadow:
```
Comment (inovizz) [inline README.md:67]: Shadow- Mehboon Subuhani

File: README.md [hunk 5]
```diff
@@ -1 +1,110 @@
...
+- Meeting:
+- Chair: [Nikhil Maan](https://pyconindia.zulipchat.com/#narrow/pm-with/231330-Nikhil-Maan)
+- Shadow: 
+- Zulip Stream: https://pyconindia.zulipchat.com/#narrow/stream/246476-wg.2Fcontent
+
+### Sponsorship
+
+- Meeting:
+- Chair: [Snehith Allamraju](https://pyconindia.zulipchat.com/#narrow/pm-with/609119-Snehith-Allamraju)
+- Shadow:
```
Comment (inovizz) [inline README.md:81]: Shadow - Abhishek Mishra

File: README.md [hunk 6]
```diff
@@ -1 +1,110 @@
...
+- Zulip Stream:
+
+### R&D (brainstorm new ideas like: poster presentation, digital swags)
+
+- Meeting:
+- Chair:
```
Comment (inovizz) [inline README.md:87]: Chair - Bhanu K

File: README.md [hunk 7]
```diff
@@ -1 +1,110 @@
...
+### Support
+
+- Meeting:
+- Chair:
+- Shadow:
+- Zulip Stream:
+
+### Code of Conduct
+
+- Meeting:
```
Comment (inovizz) [inline README.md:107]: Chair - Bhanu K

File: README.md [hunk 8]
```diff
@@ -1 +1,110 @@
...
+- Chair:
+- Shadow:
+- Zulip Stream:
+
+### Finance & Tickets
+
+- Meeting:
+- Chair:
```
Comment (inovizz) [inline README.md:94]: Chair - Ramanathan Ramakrishnamoorthy

**Review by inovizz — COMMENTED** [review]
Sorry for the delayed review. I've added some details which could be added.

**Review by inovizz — APPROVED** [review]

**Pull request merged**